apis/scheduling/v1alpha1/types.go

```diff
@@ -116,6 +119,12 @@ const (
 // +kubebuilder:resource:shortName={pg,pgs}
 // +kubebuilder:subresource:status
 // +kubebuilder:metadata:annotations="api-approved.kubernetes.io=https://github.com/kubernetes-sigs/scheduler-plugins/pull/50"
+// +kubebuilder:printcolumn:name="Phase",JSONPath=".status.phase",type=string,description="Current phase of PodGroup."
+// +kubebuilder:printcolumn:name="MinMember",JSONPath=".spec.minMember",type=integer,description="MinMember defines the minimal number of members/tasks to run the pod group."
+// +kubebuilder:printcolumn:name="Running",JSONPath=".status.running",type=string,description="The number of actively running pods."
+// +kubebuilder:printcolumn:name="Succeeded",JSONPath=".status.succeeded",type=string,description="The number of pods which reached phase Succeeded."
+// +kubebuilder:printcolumn:name="Failed",JSONPath=".status.failed",type=string,description="The number of pods which reached phase Failed."
```

Comment:
those shouldnt be of type `integer`?

Reply:
my bad.  I looked at the code and it needs to be integer. done